components/addRecipe.js

```diff
@@ -0,0 +1,56 @@
+var Component = require('choo/component')
+var html = require('nanohtml')
+
+
+module.exports = class AddRecipe extends Component {
+  constructor (name, state, emit) {
+    super(name)
+    this.state = state
+    this.emit = emit
+    this.state = state
+    this.test = this.test.bind(this)
+    // emit(state.events.ADD_RECIPE)
+    // emit(state.events.ADD_RECIPE, recipe)
+    
+}
+
+
+
+
+  update () {
+    //   const update = this.emit !== this.state.emit
+    //   this.emit = emit
+    //   return update
+  }
+
+
+  createElement () {
+      return html`
+    <div class="col-12">
+	    <form action="/" method="POST">
```

Comment:
You actually don't even need a `form` element. You are not using it. This element exists to instruct browsers on what should be posted where in an old-fashion way. However, you are not developing a classic web app, you are developing a SPA and you are using JavaScript to issue AJAX requests.

Reply:
So, instead of a form, what do I use ?